atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -98,11 +98,16 @@ public int execute(final AtlasDbServices services) {
         Map<TableReference, Optional<byte[]>> tableToStartRow = Maps.newHashMap();
 
         if ((table != null)) {
-            Optional<byte[]> startRow = Optional.of(new byte[0]);
-            if (row != null) {
-                startRow = Optional.of(decodeStartRow(row));
+            TableReference tableToSweep = TableReference.createUnsafe(table);
+            if (services.getKeyValueService().getAllTableNames().contains(tableToSweep)) {
+                Optional<byte[]> startRow = Optional.of(new byte[0]);
+                if (row != null) {
+                    startRow = Optional.of(decodeStartRow(row));
+                }
+                tableToStartRow.put(tableToSweep, startRow);
```

Comment:
Why are we using an optional here if it's _never_ null?

This would be a good opportunity to refactor this class (possibly in a separate PR). Happy to pair on this! :)